solarforecastarbiter/metrics/tests/test_deterministic.py

```diff
@@ -82,3 +82,41 @@ def test_r2(y_true, y_pred, value):
 def test_crmse(y_true, y_pred, value):
     crmse = deterministic.centered_root_mean_square(y_true, y_pred)
     assert crmse == value
+
+
+@pytest.mark.parametrize("y_true,y_pred,value", [
+    ([0, 1], [0, 1], 0.0),
+    ([1, 2], [1, 2], 0.0),
+])
```

Comment:
add `([0, 1, 2], [0, 0, 2]), 1./3.` to test cases. Hope I calculated the right output value :)

Reply:
I'm double checking the references for KSI, but I think the output for that example should be 1/6 (due to the integration of the absolute difference of the CDFs). Still working on a non-zero example for OVER and CPI.

Reply:
![ksi_test](https://user-images.githubusercontent.com/5393711/67131320-7971e400-f1c1-11e9-8d78-5eb0c0794ed2.png)

It should be 1/3.  Two problems: need to eliminate duplicate values in x, and using trapezoid integration.

Reply:
Thanks for the diagram. While that clears up the issue (duplicate values and that the trapezoid  under-estimates the integral in this case), the original citation for KSI (Espinar et al. (2009), doi:10.1016/j.solener.2008.07.009 ) states that they use trapezoid integration.

Reply:
You are correct, it does. They don't state what they mean by "unbiased estimator of the CDF" so perhaps they have to linearly interpolate between points on each CDF (trapezoid integration) whereas we're using a right-labeled ECDF, so we know the CDF values between data points.